app/helpers/needs_helper.rb

```diff
@@ -18,8 +18,64 @@ def need_categories
   end
 
   def needs
-    ['Phone triage', 'Groceries and cooked meals', 'Physical and mental wellbeing', 'Financial support',
-     'Staying Social', 'Prescription pickups', 'Book drops and entertainment', 'Dog walking']
+    needs_note_categories.sort_by { |e| e[1][:order] }.map(&:first)
+  end
+
+  def note_category_display(need_category, note_category_id)
+    needs_note_categories.find { |e| e[1][:stored_as] == need_category }
+      &.fetch(1)
+      &.fetch(:note_categories)
+      &.find { |nc| nc[:id] == note_category_id }
+      &.fetch(:display)
+  end
+
+  def needs_note_categories
+    {
+      'Phone triage' => {
+        order: 1,
+        stored_as: 'phone triage',
+        note_categories: [
+          # prefix 'phone_' makes reporting easier
+          { id: 'phone_success', display: 'Successful Phone call' },
+          { id: 'phone_failure', display: 'Failed Phone call' }
```

Comment:
There are a couple other categories, such as left message and interrupted. Siobhan should have the exact list and language.